ikea_E2201-E2213_ZHA-Z2M_control-anything.yaml

```diff
@@ -15,30 +15,36 @@ blueprint:
 
     Available controls:
 
-    - Single press **on/off** (Rodret)  **1 dot/2 dots** (Somrig) 
+    - Single press **on/off** (Rodret, Tradfri)  **1 dot/2 dots** (Somrig) 
 
-    - Double press **on/off** (Rodret, _Optional_)  **1 dot/2 dots** (Somrig)
+    - Double press **on/off** (Rodret, Tradfri, _Optional_)  **1 dot/2 dots** (Somrig)
 
-    - Hold **on/off** (Rodret)  **1 dot/2 dots** (Somrig). 
+    - Hold **on/off** (Rodret, Tradfri)  **1 dot/2 dots** (Somrig). 
     Actions will be executed every **Hold delay**, but maximum **Max number of loops** times.
   input:
     remote_device:
       name: Remote
       description: >
-        IKEA remote (Somrig, Rodret) to use.
+        IKEA remote (Somrig, Rodret, Tradfri) to use.
       default: []
       selector:
         device:
           filter:
             - integration: zha
               manufacturer: IKEA of Sweden
               model: RODRET Dimmer
+            - integration: zha     
+              manufacturer: IKEA of Sweden
+              model: TRADFRI on/off switch
             - integration: zha
               manufacturer: IKEA of Sweden
               model: SOMRIG shortcut button
             - integration: mqtt
               manufacturer: IKEA
               model: RODRET wireless dimmer/power switch
+            - integration: mqtt
+              manufacturer: IKEA
+              model: TRADFRI on/off switch (E1743)
```

Comment:
If the pattern is the same as the others, I'm pretty sure this selector will only work with z2m v1 and no longer with v2. 
Could you update it here and in the other blueprint and flag as deprectated (same as in https://github.com/damru/ha-blueprints/pull/15) ?